region/crateria/west.json

```diff
@@ -954,45 +1146,81 @@
           "from": 1,
           "to": [
             {
-              "id": 3,
-              "requires": [
-                {"or":[
-                  "canUseMorphBombs",
-                  "canUsePowerBombs"
-                ]},
-                {"obstacle":{
-                  "id": "A",
-                  "requires": [
-                    {"or":[
-                      "ScrewAttack",
-                      "canUseMorphBombs",
-                      {"canComeInCharged": {
-                        "fromNode": 1,
-                        "framesRemaining": 0,
-                        "shinesparkFrames": 90
-                      }},
-                      {"and":[
-                        "canUsePowerBombs",
+              "id": 4,
+              "strats": [
+                {
+                  "name": "Base",
+                  "notable": false,
+                  "requires": null,
+                  "obstacles": [
+                    {
+                      "id": "A",
+                      "requires": ["ScrewAttack"]
+                    },
+                    {
+                      "id": "B",
+                      "requires": ["ScrewAttack"]
+                    }
+                  ]
+                },
+                {
+                  "name": "Shinespark",
+                  "notable": false,
+                  "requires": null,
+                  "obstacles": [
+                    {
+                      "id": "B",
+                      "requires": [
+                        {"canComeInCharged": {
```

Comment:
This spark can be charged in room with a stutter 4

Reply:
That's covered. See my previous comment.